applications/ParticleMechanicsApplication/custom_strategies/strategies/MPM_residual_based_newton_raphson_strategy.hpp

```diff
@@ -15,15 +20,13 @@
 /* External includes */
 #include "boost/smart_ptr.hpp"
 
-
 /* Project includes */
 #include "includes/define.h"
 #include "includes/model_part.h"
-#include "solving_strategies/strategies/solving_strategy.h"
-#include "solving_strategies/convergencecriterias/convergence_criteria.h"
 #include "includes/kratos_flags.h"
 
-//default builder and solver
+#include "solving_strategies/strategies/solving_strategy.h"
```

Comment:
Can your derive it from the Newton Raphson f the core instead of creating one from scratch?

Reply:
Same answer as to the scheme.